### PR TITLE
docs: added cpanm installation commands and remove $ sign

### PR DESCRIPTION
**Pull request by hf400159**
### What this PR does / why we need it:

Add the cpanm installation command.
Modified etcd startup command to add log output parameters.
Remove the "$" sign before each command.

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/how-to-build.md [hunk 1]
```diff
@@ -84,33 +84,33 @@ Please refer to: [Installing Apache APISIX with Helm Chart](https://github.com/a
...
-  $ tar zxvf apache-apisix-${APISIX_VERSION}-src.tgz -C apisix-${APISIX_VERSION}
+  tar zxvf apache-apisix-${APISIX_VERSION}-src.tgz -C apisix-${APISIX_VERSION}
   ```
 
 4. Install the runtime dependent Lua libraries.
 
   ```shell
-  # Switch to the apisix-${APISIX_VERSION} directory
-  $ cd apisix-${APISIX_VERSION}
+  Switch to the apisix-${APISIX_VERSION} directory
```
Comment (zaunist) [inline docs/en/latest/how-to-build.md:108]:
```suggestion
 # Switch to the apisix-${APISIX_VERSION} directory
```
Should we need to keep the `#` ?

File: docs/en/latest/how-to-build.md [hunk 2]
```diff
@@ -185,23 +185,29 @@ Run the `apisix help` command to see the returned results and get commands and d
...
 ```shell
 # more actions find by `help`
-$ apisix help
+apisix help
 ```
 
 ## Step 4: Run Test Cases
 
-1. Install `cpanminus`, the package manager for `perl`.
+1. Install `cpanminus`, the package manager for `perl`and dependencies.
```
Comment (juzhiyuan) [inline docs/en/latest/how-to-build.md:193]:
```suggestion
1. Install `cpanminus`, the package manager for `perl` and dependencies.
```

**Review by juzhiyuan — COMMENTED** [review]
Thanks!

File: docs/en/latest/how-to-build.md (hunk 2 above)
Comment (hf400159) [reply docs/en/latest/how-to-build.md]: Remove ‘and dependencies’, because there is no installation package for this step.

**Comment by hf400159** [comment]
@bisakhmondal hi，can you review this pr? Is there anything wrong with it? thanks.

File: docs/en/latest/how-to-build.md [hunk 3]
```diff
@@ -120,12 +120,12 @@ This step is required if you have installed only Apache APISIX via RPM, Docker o
...
-$ tar -xvf etcd-v${ETCD_VERSION}-linux-amd64.tar.gz && \
-    cd etcd-v${ETCD_VERSION}-linux-amd64 && \
-    sudo cp -a etcd etcdctl /usr/bin/
-$ nohup etcd &
+ETCD_VERSION='3.4.13'
+wget https://github.com/etcd-io/etcd/releases/download/v${ETCD_VERSION}/etcd-v${ETCD_VERSION}-linux-amd64.tar.gz
+tar -xvf etcd-v${ETCD_VERSION}-linux-amd64.tar.gz && \
+  cd etcd-v${ETCD_VERSION}-linux-amd64 && \
+  sudo cp -a etcd etcdctl /usr/bin/
+nohup etcd >/tmp/etcd.log 2>&1 &
```
Comment (bisakhmondal) [inline docs/en/latest/how-to-build.md:128]: nice

File: docs/en/latest/how-to-build.md [hunk 4]
```diff
@@ -185,23 +185,29 @@ Run the `apisix help` command to see the returned results and get commands and d
...
 ```
 
 ## Step 4: Run Test Cases
 
 1. Install `cpanminus`, the package manager for `perl`.
 
+```shell
+cd /bin
+curl -L https://cpanmin.us/ -o cpanm
+chmod +x cpanm
```
Comment (bisakhmondal) [inline docs/en/latest/how-to-build.md:198]:
```suggestion
curl -L https://cpanmin.us/ -o cpanm
chmod +x cpanm
# optional
mv cpanm /bin
```
This should be a suggestion, not a strict requirement?

**Review by bisakhmondal — APPROVED** [review]
Thanks for doing this. Now the ctrl+c & ctrl+v will be a lot easier : )

**Comment by hf400159** [comment]
@bzp2010  hi，can you review this pr ? thanks

**Review by bzp2010 — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]

**Pull request closed without merging**